### ISSUE TITLE
Update Old Blog Posts from GIVcuration to GIVpower

### ISSUE THREAD

**Issue by c0ric0ri** [Giveth/giveth-planning#1105]
Example: https://medium.com/giveth/welcome-to-the-giveconomy-b3e372da63dd

We should check other past ones as well!

**Comment by c0ric0ri** [comment]
@danelsuga you can update blogs, right? Lauren is out this week.

**Comment by danelsuga** [comment]
@c0ric0ri I'm out this week too.

**Comment by danelsuga** [comment]
Did you already take care of this? I did a quick search and didn't find any others.

**Comment by c0ric0ri** [comment]
No, haven't had a chance yet. Could be that it's the only blog, yes!

**Comment by laurenluz** [comment]
Honestly i dont think we should update old blog posts without a good reason...

We can edit them with "update (date)... this has now changed to x"...maybe... but they were written at that time with information that was correct at the time.

Blogs are NOT documentation... and they should also only be updated or edited with the authors consent.

I added an edit to the giveconomy one... but yes, don't ask suga please to crtl f blogs i wrote and change their content. Especially while i am out!!! Not cool.

**Comment by laurenluz** [comment]
Sorry for overreacting @c0ric0ri!

It's a good idea to adjust things for SEO, do you think the "add an edit" approach is good enough or think it would be a lot better to omit "GIVcuration"? 

I was just a little taken aback because to me this sounded like you were asking suga to edit blogs I wrote while I'm not around... I don't think we should edit blogs without the author's consent or awareness. Anyway, I think we're on the same page now.

**Comment by c0ric0ri** [comment]
Yes I see, I was planning to discuss it with the group first, but also I did think it would be possible for Suga to edit the post with advice/review approval! Not by circumnavigating you, however. Obviously this wasn't clear since I created this issue in a hurry, working fast on Monday.

I would suggest updating GIVcuration to GIVpower everywhere, but leaving a footnote indicating that at the time of writing it was originally GIVcuration. So basically the opposite of how it is now. Maybe we put a * in front of the first GIVpower mention, and then the footnote below.

We could also add in the GIVpower video! We could improve this post in other ways as well if we think it's good. That's ok to do, and actually smarter than writing a completely new & improved version. This post is a really important one and we should be referencing it more often, so we want it to be completely up-to-speed.

It's common practice, and recommended, to update blog posts (and blog posts titles!) for SEO & user experience. Found [an article](https://www.wpglossy.com/edit-old-posts/) like this one for example that helps spell it out a bit more just as an FYI.

- Better click thru rate (updating titles from time to time helps with this)
- Fixing errors, formatting, etc
- Updating links, adding links
- Adding in keywords and more content (you've probably gained more expertise on the subject since time of writing)
